packages/components/avatar/package.json

```diff
@@ -25,6 +25,7 @@
     "@commercetools-uikit/utils": "16.7.5",
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "lodash": "4.17.21",
```

Comment:
Adding this dependency is causing the build to fail. Would you rather I just write a similar function than pulling in `startCase` from lodash?

Reply:
I think this might be because the updated lockfile has not been committed.